tools.json

```diff
@@ -13,7 +13,8 @@
   },
   "ruby_test_runner": {
     "network": "none",
-    "max_memory": "1GB"
+    "max_memory": "1GB",
+    "timeout": "10"
```

Comment:
I remembered you saying that the timeout for the Ruby test runner should be lower. Let me know if this is acceptable.

Reply:
`4` would be fine. 

Any reason not to use a string rather than a number ?